cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -59,5 +61,22 @@ func run() error {
 	if err != nil {
 		return err
 	}
-	return confgenerator.GenerateFilesFromConfig(&uc, *service, *logsDir, *stateDir, *outDir)
+	err = confgenerator.GenerateFilesFromConfig(&uc, *service, *logsDir, *stateDir, *outDir)
+	if err != nil {
+		return fmt.Errorf("The agent config file is not valid. Detailed error: %s", err)
+	}
+
+	// Only google engine service
+	if *service ==  "" { 
+		eR, err := confgenerator.GetEnabledReceivers(uc)
+		if err != nil {
+			return err
+		}
+		log.Println("Enabled Receivers", eR)
+		err = monitoring_api.SendMetricEveryInterval(eR, 60)
+		if err != nil {
+			return err
+		}
```

Comment:
Move this into a method where we can register multiple metrics at different intervals